### PR TITLE
Bugfix/0.1/porting request improvements

### PR DESCRIPTION
**Pull request by VladyslavBanul**
---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - https://app.asana.com/0/0/1202692189315167
  - https://app.asana.com/0/0/1202704574250591

### CODE REVIEW

File: src/levels/subscription/components/SubscriptionNumbersPage/TabsContent/PortingRequests/Modals/AddPortingRequestModal/Steps/Details.tsx [hunk 1]
```diff
@@ -8,18 +8,15 @@ import MultiStepForm from "storage/singletons/MultiStepForm";
 import FormInput from "components/common/Form/FormInput";
 
 import { detailsStyles } from "../styles";
-
-type defaultValuesType = {
-  rangeSize: string;
-  suggestionsAmount: string;
-};
+import FormSelect from "components/common/Form/FormSelect";
```
Comment (SergeyKoliadintsev) [inline src/levels/subscription/components/SubscriptionNumbersPage/TabsContent/PortingRequests/Modals/AddPortingRequestModal/Steps/Details.tsx:11]: change order this input please

File: src/levels/subscription/components/SubscriptionNumbersPage/TabsContent/PortingRequests/Modals/RequestInfoModal/Tabs/TabsContent/Documents.tsx [hunk 2]
```diff
@@ -18,6 +19,25 @@ type OnFileChange = (
...
+  return requiredDocuments
+    .map(el => el.allowedFormats)
+    .flat()
+    .filter((elem, index, arr) => arr.indexOf(elem) === index)
+    .reduce((res, curr, iteration, arr) => {
+      const ending =
+        iteration === arr.length - 1
+          ? ""
+          : iteration === arr.length - 2
+          ? " or "
```
Comment (SergeyKoliadintsev) [inline src/levels/subscription/components/SubscriptionNumbersPage/TabsContent/PortingRequests/Modals/RequestInfoModal/Tabs/TabsContent/Documents.tsx:32]: need add translate to thus " or " ?

File: src/levels/subscription/components/SubscriptionNumbersPage/TabsContent/PortingRequests/Modals/RequestInfoModal/Tabs/TabsContent/Documents.tsx [hunk 3]
```diff
@@ -121,12 +141,19 @@ const Documents: React.FC = () => {
...
-          requiredDocuments={requiredDocuments}
-          currentDocuments={currentDocuments}
-          onFileChange={onFileChange}
-          onFileDelete={onFileDelete}
-        />
+        <>
+          <div>
+            {t("suitable formats")}: {getSiutableFormats(requiredDocuments)}
+          </div>
+          <div>{t("max file size")}: 10MB</div>
```
Comment (SergeyKoliadintsev) [inline src/levels/subscription/components/SubscriptionNumbersPage/TabsContent/PortingRequests/Modals/RequestInfoModal/Tabs/TabsContent/Documents.tsx:148]: can you add translate to this string?

Comment (VladyslavBanul) [reply src/levels/subscription/components/SubscriptionNumbersPage/TabsContent/PortingRequests/Modals/RequestInfoModal/Tabs/TabsContent/Documents.tsx:148]: No.

Comment (SergeyKoliadintsev) [reply src/levels/subscription/components/SubscriptionNumbersPage/TabsContent/PortingRequests/Modals/RequestInfoModal/Tabs/TabsContent/Documents.tsx:149]: Okay 👎

**Pull request merged**